controllers/activemqartemis_controller_test.go

```diff
@@ -34,10 +34,11 @@ import (
 	"strconv"
 	"strings"
 
+	"time"
```

Comment:
I wonder if there is something i need to update or some other ide needs to update to get this in sync.

Reply:
that be the work of vscode :-)